src/karmen_backend/openapi/swagger.yaml

```diff
@@ -0,0 +1,1544 @@
+                state:
+                  type: string
+                  enum: [online, offline]
+            examples:
+              connect:
+                value:
+                  state: online
+              disconnect:
+                value:
+                  state: offline
+
+      responses:
+        '204':
+          description: 'OK'
+
+        '400':
+          description: >-
+            Invalid state was sent
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/api_err_response'
+              example:
+                message: <state> is an uknown state
+        '500':
+          description: >-
+            Cannot change printers state
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/api_err_response'
+              example:
+                message: Cannon change printer's connection state to <state>
+
+  '/organizations/{org_uuid}/printers/{printer_uuid}/current-job':
+    post:
+      summary: Change  state of current job
+      operationId: server.routes.printers.printer_modify_job
+      tags:
+        - printers
+      parameters:
+        - $ref: '#/components/parameters/printer_uuid_param'
+        - $ref: '#/components/parameters/organization_uuid_param'
+      requestBody:
+        content:
+          application/json:
+            schema:
+              properties:
+                action:
+                  type: string
+                  enum: [cancel, pause, resume]
+            examples:
+              cancel current job:
+                value:
+                  action: cancel
+              pause current job:
+                value:
+                  action: cancel
+              resume paused job:
+                value:
+                  action: resume
+
+      responses:
+        '204':
+          description: 'OK'
+
+        '400':
+          description: >-
+            Invalid state was sent
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/api_err_response'
+              examples:
+                Missing action:
+                  value:
+                    message: Missing action
+                No job to modify:
+                  value:
+                    message: Nothing is running
+                Generic printer error:
+                  value:
+                    message: <<generic printer error from control software>>
+
+  '/organizations/{org_uuid}/printers/{printer_uuid}/webcam-snapshot':
+    get:
+      summary: Get snapshot from webcam
+      operationId: server.routes.printers.printer_webcam_snapshot
+      tags:
+        - printers
+      parameters:
+        - $ref: '#/components/parameters/printer_uuid_param'
+        - $ref: '#/components/parameters/organization_uuid_param'
+      responses:
+        '200':
+          description: 'OK, image is returned'
```

Comment:
We should probably include the expected response content type (image/jpg or such?).

```
content:
    image/jpeg:
        schema:
            type: string
            format: binary
```